lib/repl.js

```diff
@@ -47,10 +47,11 @@ const {
   makeRequireFunction,
   addBuiltinLibsToObject
 } = require('internal/modules/cjs/helpers');
+const acorn = require('internal/deps/acorn/dist/acorn');
 const {
```

Comment:
Should these additions be placed near the other acorn work in `lib/internal/repl/`?

Reply:
I'm not sure what you mean by _these additions_ here.  The require isn't new, just the capturing of the name as `acorn` is new.  More generally, `isRecoverableError` isn't new, just completely rewritten.

Perhaps you are suggesting that `isRecoverableError` be split out into a separate file and placed into `lib/internal/repl/`?  If so, I can do that.

Reply:
I'm suggesting keeping the acorn extensions and other acorn bits closer together.